src/main/java/com/google/sps/data/Note.java

```diff
@@ -48,10 +59,33 @@ public Note(Key sessionKey, String imageUrl) throws IOException{
     public String getOriginalImageUrl(){
         return imageUrl;
     }
+
     public String getMessage(){
         return message;
     }
+    
+    public String getFilePath(){
+        return filePath;
+    }
+
+    public String getFileName(){
+        return fileName;
+    }
+
+    public void writeConvertedDoc() throws Docx4JException, IOException {
+        WordprocessingMLPackage wordMLPackage = WordprocessingMLPackage.createPackage();		
+        wordMLPackage.getMainDocumentPart().addStyledParagraphOfText("Title", "Test Notes");
+
+        wordMLPackage.getMainDocumentPart().addParagraphOfText("from docx4j!");
 
+        // saved in Team-33-Note-App/target/portfolio-1
+        fileName = String.format("Getcha_Notes_%d", sessionKey.getId());
+        filePath = String.format("%s/%s", System.getProperty("user.dir"), fileName);
+        wordMLPackage.save(new File(filePath));
+    }
+
+    // TODO: Depending on how the NLP is handled, we can avoid storing headings and sentences
```

Comment:
@joymaneetk do you agree that we can avoid storing the NLP categorized data in Datastore and simply write the converted doc from within the `Note` class?

Reply:
Should we also place the NLP data into the word doc with the notes or maybe do a separate one? Or does the word doc just contain info from NLP?